### PR TITLE
feat(widget-space): add ability to toggle composer buttons

### PR DESCRIPTION
**Pull request by moriahmaney**

### CODE REVIEW

File: packages/node_modules/@ciscospark/react-container-message-composer/src/components/ComposerButtons.test.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ describe('ComposerButtons component', () => {
 
   beforeEach(() => {
     props = {
+      composerActions: {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-container-message-composer/src/components/ComposerButtons.test.js]: can you add another test with attachFiles being false?

File: packages/node_modules/@ciscospark/react-container-message-composer/src/container.js [hunk 2]
```diff
@@ -227,6 +227,7 @@ export class MessageComposer extends Component {
     const {props} = this;
     const {
       activity,
+      composerActions,
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/react-container-message-composer/src/container.js]: Can we add to proptypes?

File: packages/node_modules/@ciscospark/react-container-message-composer/src/components/ComposerButtons.test.js (hunk 1 above)
Comment (moriahmaney) [reply packages/node_modules/@ciscospark/react-container-message-composer/src/components/ComposerButtons.test.js]: yeah i will. good catch, thanks!!

File: packages/node_modules/@ciscospark/react-container-message-composer/src/container.js (hunk 2 above)
Comment (moriahmaney) [reply packages/node_modules/@ciscospark/react-container-message-composer/src/container.js]: eagle eye! ty for catching that :)

**Pull request merged**